cli/internal/context/context.go

```diff
@@ -357,18 +357,13 @@ func (c *Context) parsePackageJSON(repoRoot turbopath.AbsoluteSystemPath, pkgJSO
 	defer c.mutex.Unlock()
 
 	if pkgJSONPath.FileExists() {
-		pkg, err := fs.ReadPackageJSON(pkgJSONPath)
+		pkg, err := fs.ReadPackageJSON(repoRoot, pkgJSONPath)
 		if err != nil {
 			return fmt.Errorf("parsing %s: %w", pkgJSONPath, err)
 		}
 
-		relativePkgJSONPath, err := repoRoot.PathTo(pkgJSONPath)
-		if err != nil {
-			return err
-		}
+		// TODO(mehulkar): Figure out if adding duplicate names to the WorkspaceGraph is a problem.
```

Comment:
Should we open up a task for this investigation?

Reply:
Will do. in practice it probably doesn't matter since the next line will throw an error, but I didn't want to change this behavior in this PR